public/src/js/utils/modify-fields-for-breaking-news.js

```diff
@@ -0,0 +1,31 @@
+import { CONST } from 'modules/vars';
+import metaFields from 'constants/article-meta-fields';
+
+function isPageBreakingNews() {
+    const url = new URL(this.location);
+    const layout = url.searchParams.get('layout') || '';
+    return url.pathname.includes('breaking-news') || layout.includes('breaking-news');
+}
+
+const metaFieldsForBreakingNews = Object.freeze(
+    metaFields.map(field => {
+        if (field.key === 'headline') {
+            return Object.assign({}, field, {
+                showCharacterCountInsteadOfCharactersLeft: true,
+                noRevertButton: true,
+                lengthWarningMessages: [
+                    [CONST.restrictedHeadlineLength, `${CONST.restrictedHeadlineLength} characters maximum`]
+                ]
```

Comment:
It feels like we could make the messaging here a little clearer. This isn't such a maximum as a suggested limit. 
Also is there any reason to represent `lengthWarningMessages` as an array?

Reply:
It was an array so we'd have the option of setting different warnings at different character counts - however, the designs send through had "90 characters maximum" at >90 and >120